wasm/argon2/build.sh

```diff
@@ -1,27 +1,43 @@
 #!/usr/bin/env bash
 
-set -e
-set -o pipefail
+set -euo pipefail
 
-# Log emcc version
-EMCC_SDK_PATH="/path/to/emsdk"
-EMCC_PATH="$EMCC_SDK_PATH/upstream/emscripten/emcc"
-$EMCC_PATH --version > emcc_version.txt
+rm -rf build/
 
-# Build start
-rm -rf dist
-mkdir dist
+touch build.log
+BUILD_LOG="$(realpath build.log)"
+echo "Built on $(date -u '+%Y-%m-%dT%H:%M:%SZ')" | tee "$BUILD_LOG"
 
-./clean-cmake.sh
-EMCC_SDK_PATH=$EMCC_SDK_PATH ARGON_JS_BUILD_BUILD_WITH_SIMD=1 ./build-wasm.sh
-mv dist/argon2.wasm ../argon2-simd.wasm
+echo "Toolchain versions" | tee -a "$BUILD_LOG"
+emcc --version | head -n1 | tee -a "$BUILD_LOG"
 
-./clean-cmake.sh
-EMCC_SDK_PATH=$EMCC_SDK_PATH ARGON_JS_BUILD_BUILD_WITH_SIMD=0 ./build-wasm.sh
-mv dist/argon2.wasm ../argon2.wasm
+# FIXME: Redownload the source if argon2 ever has source updates. At the time of writing it was last changed 5 years ago so this is probably not a high priority.
```

Comment:
Could you add a comment to the repository where the source files are coming from? Is it https://github.com/P-H-C/phc-winner-argon2?